common/lib/common-definitions/api-report/common-definitions.api.md

```diff
@@ -196,11 +196,11 @@ export interface ILoggingError extends Error {
 }
 
 // @public
-export interface ITaggedBaseTelemetryPropertyType {
+export interface ITaggedBaseTelemetryProperty {
```

Comment:
Sorry but on second thought can we put `Value` at the end of this too?  It's a tagged value - the "property" is the key-value pair (like the `Properties` type)
```suggestion
export interface ITaggedBaseTelemetryPropertyValue {
```